template/{% if AddPrecommit %}.pre-commit-config.yaml{% endif %}.jinja

```diff
@@ -1,49 +1,50 @@
+# This file was created by BestieTemplate.jl. Create issues there: https://github.com/abelsiqueira/BestieTemplate.jl/issues
```

Comment:
Hi @lyashevska, these files have all lost their indentation, can you fix it, please?

Reply:
Thanks for spotting this, will check now. It must be my editor.